certify-service/src/main/java/io/mosip/certify/utils/DIDDocumentUtil.java

```diff
@@ -27,32 +27,24 @@
 import io.mosip.certify.core.exception.CertifyException;
 import lombok.extern.slf4j.Slf4j;
 import org.bouncycastle.util.BigIntegers;
+import org.springframework.beans.factory.annotation.Autowired;
 
 @Slf4j
 public class DIDDocumentUtil {
 
     private static final String MULTICODEC_PREFIX = "ed01";
 
-    public static Map<String, Object> generateDIDDocument(String vcSignAlgorithm, String certificateString, String issuerURI, String issuerPublicKeyURI) {
-
-        HashMap<String,Object> didDocument = new HashMap<String,Object>();
-        didDocument.put("@context", Collections.singletonList("https://www.w3.org/ns/did/v1"));
-        didDocument.put("alsoKnownAs", new ArrayList<>());
-        didDocument.put("service", new ArrayList<>());
-        didDocument.put("id", issuerURI);
-        didDocument.put("authentication", Collections.singletonList(issuerPublicKeyURI));
-        didDocument.put("assertionMethod", Collections.singletonList(issuerPublicKeyURI));
-
+    public static Map<String, Object> generateVerificationMethod(String vcSignAlgorithm, String certificateString, String issuerURI, String issuerPublicKeyURI, String kid) {
```

Comment:
Since you got the kid now, can we not remove issuerPublicKeyURI ? Is there used for anything ?